test/Interop/Cxx/namespace/free-functions-irgen.swift

```diff
@@ -1,4 +1,5 @@
 // RUN: %target-swift-emit-ir -I %S/Inputs -enable-cxx-interop %s | %FileCheck %s
+// REQUIRSE: SR15759
```

Comment:
This only fails on windows. Please just disable it on windows.

Reply:
I think that this issue really should be investigated - this seems to be occurring with the C++ interop tests, not the other ones, so I think that we need to understand why this is only impacting the C++ interop.  Maintaining parity is important, especially since libswift is aggressively using this functionality.  At the moment, Im inclined to say that this should be global.

Reply:
Please don't disable this on non-windows bots. This is important functionality to test. 

I suspect this one is causing more issues because there are a lot of (expensive) decls here so it probably hits the race condition more. I don't really know, though.